20/Dockerfile

```diff
@@ -23,11 +23,11 @@ RUN set -xe \
 	&& tar -xzf otp-src.tar.gz -C $ERL_TOP --strip-components=1 \
 	&& rm otp-src.tar.gz \
 	&& ( cd $ERL_TOP \
-	  && ./otp_build autoconf \
-	  && gnuArch="$(dpkg-architecture --query DEB_BUILD_GNU_TYPE)" \
-	  && ./configure --build="$gnuArch" \
-	  && make -j$(nproc) \
-	  && make install ) \
+	&& ./otp_build autoconf \
+	&& gnuArch="$(dpkg-architecture --query DEB_BUILD_GNU_TYPE)" \
+	&& ./configure --build="$gnuArch" \
+	&& make -j$(nproc) \
+	&& make install ) \
```

Comment:
could you talk why need this? the Line25 was `( cd $ERL_TOP` in a subshell, following lines I want to keep them indented further;
this is a trick in shell programming: you can do something change directory in a subshell, after end of subshell, you get your original working directory; in the docker case, the default working directory is root `/`